MvvmCross/Platforms/Android/Binding/Views/MvxFilteringAdapter.cs

```diff
@@ -148,6 +156,35 @@ public override Object GetItem(int position)
             return base.GetItem(position);
         }
 
+        public override object GetRawItem(int position)
+        {
+            lock(_syncLock)
+            {
+                var element = FilteredItemsSource?.ElementAt(position);
+                return element;
+            }
+        }
+
+        public override int GetPosition(object item)
+        {
+            lock (_syncLock)
+            {
+                var pos = FilteredItemsSource?.GetPosition(item) ?? 0;
```

Comment:
Should default be 0 or -1

Reply:
Hmm, this is used in `GetItem` and `GetRawItem` to get the element. Not sure what the appropriate default value should be here. If it is 0 and the ItemsSource contains 0 items you will get a Index error, the same with -1. However, if it contains items 0 could return a wrong item and -1 would still throw a Index error. Not sure what is preferred here.